ktor-server/ktor-server-plugins/ktor-server-thymeleaf/jvm/test/io/ktor/server/thymeleaf/ThymeleafTest.kt

```diff
@@ -194,6 +194,54 @@ class ThymeleafTest {
         }
     }
 
+    @Test
+    fun testFragmentReturn() {
+        withTestApplication {
+            println("Test fragments")
```

Comment:
Please remove `println` here and below

Reply:
Thanks for your review! I will remove them